### PR TITLE
[TW-761] Small changes to scr

### PR DESCRIPTION
**Pull request by petewitcher**

### CODE REVIEW

File: src/pages/docs/running-collections/scheduling-collection-runs.md [hunk 1]
```diff
@@ -144,27 +144,27 @@ Many Postman users use collections as repositories of requests, which they run i
...
 
 1. Under **RUN ORDER**, change the order by dragging and dropping requests. Skip a request by unchecking its checkbox.
 
     ![Custom run order v10](https://assets.postman.com/postman-docs/v10/scr-custom-order-v10.gif)
 
-1. Select the **Save Changes** button.
+1. Select the **Schedule Run** button.
 
-    When a new request is added or deleted from a scheduled run's collection, the scheduled run has a **Review Changes** link next to it under **Runs > Scheduled runs**. Click the link to review and edit the changes. In the example below, a new request was added:
+When a new request is added or deleted from a scheduled run's collection, the scheduled run has a **Review Changes** link next to it under **Runs > Scheduled runs**. Click the link to review and edit the changes. In the example below, a new request was added:
```
Comment (mark-anthony-postman) [inline src/pages/docs/running-collections/scheduling-collection-runs.md:163]:
```suggestion
When a new request is added or deleted from a scheduled run's collection, the scheduled run has a **Review Changes** link next to it under **Runs > Scheduled runs**. Select the link to review and edit the changes. In the example below, a new request was added:
```

File: src/pages/docs/running-collections/scheduling-collection-runs.md [hunk 2]
```diff
@@ -144,27 +144,27 @@ Many Postman users use collections as repositories of requests, which they run i
...
 
-    ![Schedule runs](https://assets.postman.com/postman-docs/v10/scr-review-custom-order-v10.gif)
+![Schedule runs](https://assets.postman.com/postman-docs/v10/scr-review-custom-order-v10.gif)
 
-    > Requests run as part of a scheduled run are rate limited and usage is deducted from your maximum Monitoring Usage. For more information about checking your monitor usage, see [Managing monitor usage](/docs/monitoring-your-api/monitor-usage/).
+> Requests run as part of a scheduled run are rate limited and usage is deducted from your maximum Monitoring Usage. For more information about checking your monitor usage, see [Managing monitor usage](/docs/monitoring-your-api/monitor-usage/).
```
Comment (mark-anthony-postman) [inline src/pages/docs/running-collections/scheduling-collection-runs.md:167]:
```suggestion
> Requests run as part of a scheduled run are rate limited and usage is deducted from your maximum monitor usage. For more information about checking your monitor usage, see [Managing monitor usage](/docs/monitoring-your-api/monitor-usage/).
```

**Review by mark-anthony-postman — APPROVED** [review]
Couple of minor comments, otherwise LGTM! 😀

File: src/pages/docs/running-collections/scheduling-collection-runs.md (hunk 2 above)
Comment (petewitcher) [reply src/pages/docs/running-collections/scheduling-collection-runs.md:167]: Ack how did I miss that. Thanks Mark!

**Pull request merged**